index.html

```diff
@@ -146,19 +146,20 @@ <h1>The Registry</h1>
           Ed25519SigningKey
       </td>
       <td>
-          None
+          <a href="https://w3c-dvcg.github.io/lds-ed255192018/">Ed25519 Signature Suite 2018</a>
       </td>
       <td>
-          Kim Duffy
+          Markus Sabadello
       </td>
       <td>
 <pre>
 {
-  "id": "did:example:123456789abcdefghi/keys/2",
-  "type": "Ed25519SigningKey",
-  "owner": "did:example:123456789abcdefghi",
-  "expires": "2017-02-08T16:02:20Z",
-  "publicKeyBase64": "IOmA4R7TfhkYTYW87...CBMq2/gi25s="
+  "@context": ["https://w3id.org/security/v1"],
+    "id": "did:example:123456789abcdefghi#keys-1",
+    "type": "Ed25519VerificationKey",
+    "owner": "did:example:123456789abcdefghi",
+    "publicKeyBase64" : "H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
```

Comment:
`publicKeyBase64` is a base64-encoding of what format specifically?

Reply:
Ed25519 keys are simply 32 raw bytes that don't have a "format".
But actually I think I made a mistake in this example, the value string is in Base58 encoding, not Base64. So I should either change the value string, or use "publicKeyBase58" instead of "publicKeyBase64"....

Reply:
https://blog.mozilla.org/warner/2011/11/29/ed25519-keys/

Ok, since in this case the data is *literally* the raw public key bytes, so I think the name is appropriate (with the base58 fix you mentioned).

Reply:
Generally if you just use base64 or hex we should be talking about the raw bytes with no specific structure like ASN.1. Can we make sure that that is clear somewhere in a notes section or referenced somewhere?